frontend/packages/core/src/card.tsx

```diff
@@ -69,7 +58,28 @@ const CardHeader = ({ avatar, children, title }: CardHeaderProps) => (
   </StyledCardHeaderContainer>
 );
 
-interface CardContentProps extends MuiCardContentProps {}
+// Material UI Spacing system supports many props https://material-ui.com/system/spacing/#api
+// We can add more to this list as use cases arise
+interface SpacingProps {
+  padding?: number;
+  // shorthand for padding
+  p?: number;
+}
+
+const BaseCardContent = styled.div<SpacingProps>`
+  ${spacing}
+`;
```

Comment:
lmk your thoughts on this pattern. saw it here https://dev.to/andrecastelo/prop-forwarding-with-emotion-material-ui-and-material-ui-system-40kd and here https://github.com/resir014/chungking/blob/9a03099f5ddcee79216b927958872d4a49ffe03b/packages/chungking-react/src/layout/inline/Inline.tsx#L17-L19. it was tricky to figure out how the spacing system can be used in any component b/c Material UI docs only show it being used in the `Box` component which by default has `spacing` https://material-ui.com/system/spacing/

Reply:
This is great!